great_expectations/metrics/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .batch.row_count import BatchRowCount
-from .column_values.between import ColumnValuesBetween
+from .column_values.non_null import ColumnValuesNonNull, ColumnValuesNonNullCount
```

Comment:
Why are we deleting between? Isn't this a new metric in addition to between?

Reply:
Between was there to demonstrate how it would all look, but then I learned we needed to add this (different) metric to support another epic. Between was never tested, which is really the bulk of the remaining work at this point.